src/main/kotlin/de/bund/digitalservice/useid/SessionController.kt

```diff
@@ -0,0 +1,24 @@
+package de.bund.digitalservice.useid
+
+import de.bund.digitalservice.useid.model.ClientRequestSession
+import de.bund.digitalservice.useid.model.ClientResponseTCTokenUrl
+import de.bund.digitalservice.useid.service.TCTokenUrlService
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/api/v1")
+class SessionController(private val tcTokenUrlService: TCTokenUrlService) {
+    @PostMapping(
+        path = ["/init-session"],
```

Comment:
I don't like this name with a verb and `-` 😅 But I don't really have a better idea... Maybe just `session`? Or `identification`?

Reply:
I like `session` better!